src/color.rs

```diff
@@ -669,14 +669,17 @@ pub enum NumberOrPercentage {
 }
 
 impl NumberOrPercentage {
-    fn unit_value(&self) -> f32 {
+    /// Return the value as a percentage.
+    pub fn unit_value(&self) -> f32 {
```

Comment:
I think in the calc() code we call this `unitless_value`, maybe do the same for consistency?

Reply:
`unit_value` comes from `Token::Percentage { unit_value, .. }`. Consistency can go either way here, so without having to swap the two methods around, because unitless value will now be the "other" one, I'd rather keep it as is for now.